### ISSUE TITLE
False positive to add "symfony/var-dumper" to "dev packages"

### ISSUE THREAD

**Issue by theofidry** [kalessil/phpinspectionsea#1910]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | PHP Inspections (EA Extended) 5.0.0.0 |
| Language level | 8.1  |

Current behaviour
--

When adding `symfony/var-dumper` as a **required** package, you will get the following warning in the `composer.json`:

```
[EA] Dev-packages have no security guaranties, invoke the package via require-dev instead. 
```

Expected behaviour
--

When I asked Nicolas Grekas (the author of the package) about it, he told me there is nothing dev specific about this package (except probably it's first use case when being developed) and there should not be anything that would present a risk on production.

Environment details
---

Information from Help - About dialog
